iosxr_iso2vbox.py

```diff
@@ -65,38 +63,295 @@
 import time
 import subprocess
 import getpass
-import shlex
 import argparse
 from argparse import RawDescriptionHelpFormatter
 import re
+import logging
+import pexpect
 
 # Telnet ports used to access IOS XR via socat
-console_port = 65000
-aux_port = 65001
+CONSOLE_PORT = 65000
+AUX_PORT = 65001
 
+logger = logging.getLogger(__name__)
 
-def run(command_string, debug=False):
+
+def set_logging():
+    '''
+    Set basic logging format.
+    '''
+    FORMAT = "[%(filename)s:%(lineno)s - %(funcName)20s() ] %(message)s"
```

Comment:
It put a little gap between the function name and the closing paren - e.g:

rwellum@RWELLUM-M-34DF:[~/Desktop/Boxes]: iosxrv-x64-vbox/iosxr_iso2vbox.py iosxrv-fullk9-x64.iso
[iosxr_iso2vbox.py:430 -                 main() ] Creating Vagrant VirtualBox
[iosxr_iso2vbox.py:155 -         configure_xr() ] Logging into Vagrant Virtualbox and configuring IOS XR
[iosxr_iso2vbox.py:586 -                 main() ] Powering down and generating Vagrant VirtualBox
[iosxr_iso2vbox.py:616 -                 main() ] Created: /Users/rwellum/Desktop/Boxes/machines/iosxrv-fullk9-x64/iosxrv-fullk9-x64.box
[iosxr_iso2vbox.py:626 -                 main() ] Running basic unit tests on Vagrant VirtualBox...
[iosxr_iso2vbox.py:647 -                 main() ] Passed basic test, box /Users/rwellum/Desktop/Boxes/machines/iosxrv-fullk9-x64/iosxrv-fullk9-x64.box is sane

But I think it's just as clear to remove that gap.

Example of the verbose - without the gap:

rwellum@RWELLUM-M-34DF:[~/Desktop/Boxes]: iosxrv-x64-vbox/iosxr_iso2vbox.py iosxrv-fullk9-x64.iso -v
[iosxr_iso2vbox.py:418 -                 main()] Input ISO is iosxrv-fullk9-x64.iso
[iosxr_iso2vbox.py:426 -                 main()] iosxrv-fullk9-x64.iso is a full image, RAM allocated is 4096 MB
[iosxr_iso2vbox.py:430 -                 main()] Creating Vagrant VirtualBox
[iosxr_iso2vbox.py:101 -                  run()] command: 'VBoxManage -v'

[iosxr_iso2vbox.py:106 -                  run()] Command succeeded with code 0:
[iosxr_iso2vbox.py:108 -                  run()] Output for: VBoxManage -v
[iosxr_iso2vbox.py:109 -                  run()] 5.0.26r108824

[iosxr_iso2vbox.py:433 -                 main()] Virtual Box Manager Version: 5.0.26r108824